packages/evershop/src/modules/stripe/pages/frontStore/checkout/Stripe.jsx

```diff
@@ -18,10 +18,19 @@ const stripeLoader = (publishKey) => {
 };
 
 function StripeApp({ stripePublishableKey }) {
+  const options = {
+    mode: 'payment',
+    currency: 'eur',
+    amount: 1337,
+	fields: {
+		billingDetails: 'never',
+	}
```

Comment:
Hi @ultimate-tester ,

Whoops. We can not hardcoded the amount and currency like this. It must be from the cart

Reply:
Would you be able to help me with this? My experience with React is limited so I don't have a clue on how to add these fields into context :(

Reply:
Sure, I will take care of it. Please give me sometime